adapter/internal/operator/synchronizer/synchronizer.go

```diff
@@ -295,7 +314,7 @@ func SendEventToPartitionServer() {
 			organization := api.APIDefinition.Spec.Organization
 			version := api.APIDefinition.Spec.APIVersion
 			apiName := api.APIDefinition.Spec.APIDisplayName
-			apiUUID := string(api.APIDefinition.GetUID())
+			apiUUID := string(api.APIDefinition.Name)
```

Comment:
please use the namespacedName, so the namespace also included

Reply:
According to a offline discussion I had with @tharindu1st this seems to be fine for partition usecases. hence resolving this issue